plugins/elan-ts/fu-elan-ts-hid-device.c

```
@@ -0,0 +1,710 @@
+ *
+ * An ELAN Touchscreen HID device.
+ */
+struct _FuElanTsHidDevice {
+	FuHidrawDevice parent_instance;
+	FuElanTsState touch_state;
+	guint16 bc_version;
+	guint16 fw_id;
+	guint16 fw_version;
+	guint16 test_version;
+};
+
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If G_DECLARE_FINAL_TYPE is a macro then please configure it.

(unknownMacro)